.github/workflows/regenerate_integration_tests.yml

```diff
@@ -17,15 +17,15 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v5
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: "3.11"
```

Comment:
this fix makes me wonder how it ever worked 😆

Reply:
It didn't lol Only Robert and I have made use of it thus far, so for now we put the fix in our branches. :)